doc/gestor_tareas.md

```diff
@@ -0,0 +1,33 @@
+# Gestor de tareas
+
+Un gestor de tareas es una herramienta que nos ayuda a realizar tareas repetitivas de manera más amena.
+
+## Criterios de decisión
+
+Para la elección de nuestro gestor de tareas tenemos que tener en cuenta diferentes criterios, que son importantes para tomar la mejor decisión. Estos criterios son:
+
+- **Comunidad:** El gestor de dependencias debe tener una comunidad que pueda ayudar a resolver problemas.
+
+- **Mantenimiento**: La herramienta es conveniente que tenga cierta antigüedad, para demostrar que es solida, pero que tenga ultimas versiones, que sean lo más recientes posibles.  
+
+
+## Gestores de tareas
+
+Existen muchos gestores de tareas que podemos utilizar, entre ellos se encuentran: Make, Broccoli y NPM scripts, aunque hay muchos más.
+
+### Make
+
+Make es un gestor de tareas que tiene una gran comunidad que puede ayudar a resolver problemas. Además de ser simple y eficiente, Make es un gestor de tareas que es fácil de configurar y no requiere instalar herramientas adicionales. 
+
+
+### Broccoli
```

Comment:
Conviene que pongáis siempre un enlace. En algunso casos se puede saber de qué estáis hablanddo, en otros no. En [este](https://github.com/broccolijs/broccoli) el último commit es de hace más de dos años y el flujo de test falla. 
Si usas Snyk Advisor (que os lo mencioné en clase), ¿qué te dice sobre su salud?

Reply:
:arrow_up: No has visto esto o no has hecho nada al respecto.

Reply:
Si lo había visto, pensaba que lo que se refería era que lo siguiese indicando, pero que añadiese que era mala elección, por eso al final he comentado que su mantenimiento es malo, ya que hace más de dos años que no tiene ningún tipo de soporte, por lo tanto, que debo hacer ¿eliminarlo? o ¿o lo dejo como esta que he indicado que no es buena opción por su mal mantenimiento?

Reply:
1. Tienes que cambiar o cerrar *todos* los comentarios
2. Si su mantenimiento es malo, ¿habrá otros que tenga un mantenimiento bueno que merezca la pena listar *en vez de este?